packages/cli/src/commands/index.ts

```diff
@@ -22,6 +22,16 @@ export function registerCommands(program: CommanderStatic) {
     .command('app:build')
     .description('Build an app for a production release')
     .option('--stats', 'Write bundle stats to output directory')
+    .option(
+      '--parallel [value]',
+      'Enable multi-process parallel running and set number of concurrent runs',
+      undefined,
```

Comment:
Default to `true`?

Reply:
I'll try again but I think when I did that commander won't properly number values and Boolean flags for the same flag.

Reply:
Could do it through the description too, just wanna make it clear that it's not a flag you need to use unless you do

Reply:
I'll have a look at it again. I started this a few weeks back so maybe my memory is a bit fuzzy.

Reply:
@Rugvip - So yes, I could set `true` for `--parallel` & `false` for `--no-parallel` and it will work fine. However, I now remember that I set this to undefined, because I wanted a way to know when the flag wasn't provided at all in order to ignore the `terser` setup in the `optimization` module. If we provide default values there is no easy way to know if the option was explicitly set or not. That said, if we don't care and always want to configure the `terser` settings (which I believe are also default when omitted) then this isn't really a concern.

Reply:
It's parallel by default right? So `true` -> no explicit configuration

Reply:
I think so yeah. So what your saying is omit when true, and add the terser config otherwise?

Reply:
Yes, as long as we know it's the exact same config as the one webpack would add. Thinking about that it would actually make more sense to always add the config, but `true` should reflect what webpack already does. Just wanna avoid build output differing by the parallel setting because it ends up using different versions of terser

Reply:
I'm pretty sure I've matched the `terser-webpack-plugin` version to the default for webpack. Updated to only include the config to only run when using `parallel` other then `true`. I think this is probably the safest to begin with. We can potentially make it the default later pretty easily.

Reply:
On second thought, keeping this undefined by default makes it much easier to reason about using an environment variable as a fallback, which seems like the appropriate precedence (cli flag -> env variable -> default value)